js/models/conversations.js

```diff
@@ -1107,13 +1124,43 @@
       // eslint-disable-next-line no-param-reassign
       message.quotedMessageFromDatabase = queryMessage;
 
-      this.forceRender(message);
       return true;
     },
+    async loadQuotedMessage(message, quotedMessage) {
+      // eslint-disable-next-line no-param-reassign
+      message.quotedMessage = quotedMessage;
```

Comment:
If this is new code, could we treat arguments as immutable for easier testing?

Reply:
As we discussed on Friday, this is the only way we can send data from here ('action creator') to the MessageView ('selector'). I don't like hanging information off of the Message model like this, but we have very few options for pipelining data as needed.